apps/microtvm/arduino/example_project/project.ino

```diff
@@ -0,0 +1,10 @@
+#include "src/model.h"
+
+void setup() {
+  TVMInitialize();
+  //TVMExecute(input_data, output_data);
```

Comment:
We need to keep it commented, as otherwise `project.ino` won't compile since `input_data` and `output_data` aren't defined. We can't define those either, because until input and output metadata is re-added to MLF, we don't know the input and output sizes and data types.

Reply:
can you add this in a comment above the commented TVMExecute?

Reply:
`project.ino` will be the file a non-technical user sees, so I'm not sure I want to use it to document too much of the internal workings of TVM. Where else could we put that information?